src/trace.rb

```diff
@@ -265,8 +278,7 @@ def load_variables(binding)
 $tracer.ignore('gems/')
   
 
-trace_args = ARGV
-ARGV = ARGV[1..-1]
+trace_args = [program] + ARGV
```

Comment:
does this work? it was important to override ARGS, if the user program uses it directly IIRC

Reply:
ok, sorry: `ARGV.shift` does it now , however if we have more args like `-o`, maybe we need to move past them as well(but i assume `order!` does that)

Reply:
The tests pass, so it should work :)

Reply:
they don't seem to test for `ARGV` usage

Reply:
Right, I'll add such test in a follow-up PR.